views/display-form-addViaCode.twig

```diff
@@ -0,0 +1,49 @@
+{#
+/**
+* Copyright (C) 2019 Xibo Signage Ltd
+*
+* Xibo - Digital Signage - http://www.xibo.org.uk
+*
+* This file is part of Xibo.
+*
+* Xibo is free software: you can redistribute it and/or modify
+* it under the terms of the GNU Affero General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* any later version.
+*
+* Xibo is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU Affero General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Xibo.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#}
+
+{% extends "form-base.twig" %}
+{% import "forms.twig" as forms %}
+
+{% block formTitle %}
+    {% trans "Add Display via Code" %}
+{% endblock %}
+
+{% block formButtons %}
+    {% trans "Help" %}, XiboHelpRender("{{ help }}")
+    {% trans "Cancel" %}, XiboDialogClose()
+    {% trans "Save" %}, $("#displayAddViaCodeForm").submit()
+{% endblock %}
+
+{% block formHtml %}
+    <div class="row">
+        <div class="col-md-12">
+            <form id="displayAddViaCodeForm" class="XiboForm form-horizontal" method="post" action="{{ urlFor("display.addViaCode") }}">
+
+                {% set title %}{% trans "Code" %}{% endset %}
+                {% set helpText %}{% trans "Please provide the code displayed on the Player screen" %}{% endset %}
+                {{ forms.input("user_code", title, "", helpText) }}
+
```

Comment:
Would it be sensible here to show an info box explaining what happens when they submit the form, and that their CMS will need access to the Authentication Service for this to work?